src/Rel8/Type.hs

```diff
@@ -128,10 +128,10 @@ instance DBType Int64 where
 instance DBType Float where
   typeInformation = TypeInformation
     { encode = \x -> Opaleye.ConstExpr
-        if | x == (1 /0) -> Opaleye.OtherLit "'Infinity'"
-           | isNaN x     -> Opaleye.OtherLit "'NaN'"
-           | x == (-1/0) -> Opaleye.OtherLit "'-Infinity'"
-           | otherwise   -> Opaleye.NumericLit $ realToFrac x
+        if | x == (1 / 0) -> Opaleye.OtherLit "'Infinity'"
+           | isNaN x      -> Opaleye.OtherLit "'NaN'"
+           | x == (-1 / 0) -> Opaleye.OtherLit "'-Infinity'"
+           | otherwise     -> Opaleye.NumericLit $ realToFrac x
```

Comment:
```suggestion
        if | x == (1 / 0)  -> Opaleye.OtherLit "'Infinity'"
           | isNaN x       -> Opaleye.OtherLit "'NaN'"
           | x == (-1 / 0) -> Opaleye.OtherLit "'-Infinity'"
           | otherwise     -> Opaleye.NumericLit $ realToFrac x
```